<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_detail</class>
 <widget class="QWidget" name="config_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Config Details</string>
  </property>
  <widget class="QTextEdit" name="location">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="url_name">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roiBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Roi Selection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lineBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Selection</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>491</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="imageScreen"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
